Tests/Mock/CategoryInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ClassificationBundle\Model;
```

Comment:
yes, this is a test to Mock if the file CategoryInterface from ClassificationBundle exists